esp/esdllib/esdl_def.cpp

```diff
@@ -1144,6 +1145,11 @@ class EsdlDefService : public EsdlDefObject, implements IEsdlDefService
         }
     }
 
+    const char * queryStaticNamespace()
+    {
+        return props->queryProp("staticnamespace");
```

Comment:
Does the simpler property name "namespace" already mean something else?  Also wondering if "xmlns" is better since other types of namespaces might be useful in the future and this only covers xml.

Reply:
the "static" is useful, almost necessary to illustrate the behavior of this entry which is to overwrite the dynamic generation of the ns. (no version, no method, no lowercase service)
 
xmlns seems fine, the only counter argument is that we have a basenamespace configuration entry we support. That one allows admins to set the base xmlns, but we continue to attach the ver, method,etc. let me know what you think.

Reply:
Consistency is key.  If we use basenamespace now, then staticnamespace is fine.